modules/sap_hana_svpc/sap_hana_python/main.tf

```diff
@@ -0,0 +1,32 @@
+data "http" "sap-hana-py" {
```

Comment:
This module is identical to `sap_hana_ha_svpc/sap_hana_python`; it should defined once directly under `modules` and used by `sap_hana_svpc` and `sap_hana_ha_svpc`. It does not require a README since it is not public.

Reply:
its defined only once now